docs/examples/create_attribute.rst

```diff
@@ -189,11 +187,8 @@ You can query the attribute to get any of the values, and even add an observer a
 
 .. note::
 
-    It is not possible to reliably use this approach to create independent modules because:
-
-    * Only one MAVLink callback can be set at a time in the running program (``set_mavlink_callback()``) 
-    * There is no reliable way to import modules without adding them properly to the Python environment.
-
+    It is not possible to reliably use this approach to create independent modules because only one 
```

Comment:
Still correct for now - because this describes `set_mavlink_callback`. If/when there is a better way to handle messages (and yes being able to define an arbitrary number handlers and register them as observers of messages would be cool). I'll revisit when new functionality in place.